src/commands/fmt.ts

```diff
@@ -0,0 +1,135 @@
+import { Command, flags } from '@oclif/command';
+import * as Parser from '@oclif/parser';
+import { format } from '@src/formatter';
+import {
+  DiscardSurroundingTextBehaviour,
+  discardSurroundingText
+} from '@src/formatter';
+import { parseTFFileContents } from '@src/parser';
+import { printTFFileContents } from '@src/printer';
+import dedent from 'dedent';
+import * as disparity from 'disparity';
+import { promises as fs } from 'fs';
+
+export default class Fmt extends Command {
+  public static description = 'Formats the given input';
+
+  public static examples = [`$ terra fmt <file>`];
+
+  public static flags = {
```

Comment:
I'm super keen to get feedback on `flags`, and their `char` values: I've not worked with Linux long enough to be confident in knowing what the "common" short chars are (i.e `r` for recursive (except that `chmod` & `chgrp` use `-R` 🙄), `f` for `force`).